src/spec/create_extension_spec.py

```diff
@@ -1,47 +1,394 @@
+                                                                             'running index, Columns hold the different entry names, Layers (up to nine) hold '
+                                                                             'the headstage dependent data in the first 8 and the headstage independent data in the 9th layer. '
+                                                                             'See also https://alleninstitute.github.io/MIES/labnotebook-docs.html.',
+                                                                             neurodata_type_inc='LabNotebookNumericalValues'
+                                                                             ),
+                                                              NWBDatasetSpec(doc='Numerical labnotebook keys: First row is the name, second row is the unit and third row is the tolerance. '
+                                                                             'Columns are the running index. '
+                                                                             'See also https://alleninstitute.github.io/MIES/labnotebook-docs.html.',
+                                                                             neurodata_type_inc='LabNotebookNumericalKeys'
+                                                                             ),
+                                                              NWBDatasetSpec(doc='Textual labnotebook keys: First row is the name, second row is the unit and third row is the tolerance. '
+                                                                             'Columns are the running index. '
+                                                                             'See also https://alleninstitute.github.io/MIES/labnotebook-docs.html.',
+                                                                             neurodata_type_inc='LabNotebookTextualKeys'
+                                                                             ),
+                                                              NWBDatasetSpec(doc='Textual labnotebook values: Rows are the '
+                                                                             'running index, Columns hold the different entry names, Layers (up to nine) hold '
+                                                                             'the headstage dependent data in the first 8 and the headstage independent data in the 9th layer. '
+                                                                             'See also https://alleninstitute.github.io/MIES/labnotebook-docs.html.',
+                                                                             neurodata_type_inc='LabNotebookTextualValues'
+                                                                             )
+                                                              ]
```

Comment:
These lines are already in the NWBGroupSpec definition for the neurodata_type LabNotebookDevice on line 121.

```suggestion
```